commit e9c0ea68a0eb127dfc0ad74503e903e945641ab5 (refs/changes/63/3963/1)
Author: Mike Zeller <mike.zeller@joyent.com>
Date:   2018-05-16T14:33:22-07:00 (1 year, 5 months ago)
    
    TRITON-348 breaks setting up fabrics when sdc_underlay_assignment is set to manual

diff --git a/lib/post-setup/fabrics.js b/lib/post-setup/fabrics.js
index 86ffa77..6af80f1 100644
--- a/lib/post-setup/fabrics.js
+++ b/lib/post-setup/fabrics.js
@@ -631,6 +631,7 @@ Fabrics.prototype.checkAssignment = function checkAssignment(cb) {
             return;
         }
         cb(null);
+        return;
     }
 
     self.sdcadm.napi.getNetworkPool(self.data.sdc_underlay_pool,
